### PR TITLE
MF1933050

### PR DESCRIPTION
**Pull request by lgyhhh**

### CODE REVIEW

File: MF1933050/java/Calendar_before_Object.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+import java.util.Calendar;
+
+public class Calendar_before_Object {
+    /**
+     * input: 2
+     * class0 class0="2019.1.2 3:4:5"
+     * when when="2019.1.2 3:4:6"
+     * output: 1
+     * ret0 ret0=true
+     */
```
Comment (idiomaticrefactoring) [inline MF1933050/java/Calendar_before_Object.java]: 变量初始化问题

File: MF1933050/java/File_toPath.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import java.nio.file.Path;
+
+public class File_toPath {
+
+    /**
+     * input: 1
+     * class0 class0= new File("/Users/lgy/Desktop")
+     * output: 1
+     * ret0 ret0="/Users/lgy/Desktop"
+     */
```
Comment (idiomaticrefactoring) [inline MF1933050/java/File_toPath.java]: class0,ret0初始化问题

File: MF1933050/java/Calendar_before_Object.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        System.out.println(ret0);
+    }
+
+    /**
+     * input: 2
+     * class0 class0="2019.1.2 3:4:5"
+     * when when="2019.1.2 3:4:5"
+     * output: 1
+     * ret0 ret0=false
+     */
```
Comment (idiomaticrefactoring) [inline MF1933050/java/Calendar_before_Object.java]: 变量初始化问题

**Comment by lgyhhh** [comment]
String, 262 same
Integer, 82 change
Calendar, 98 same
File, 210 change
PrintStream, 146 change

**Comment by lgyhhh** [comment]
String, 262 same
Integer, 82 change
Calendar, 98 same
File, 210 change
PrintStream, 146 change

**Review by idiomaticrefactoring — APPROVED** [review]
perfect

**Comment by TonyLee12** [comment]
MF1933053李志成评估已通过

**Pull request merged**